activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -276,7 +277,13 @@ def _active_record_fixture(fixture_set_name, *fixture_names)
           raise StandardError, "No fixture set named '#{fixture_set_name.inspect}'"
         end
       end
-      alias_method :fixture, :_active_record_fixture
+
+      # Generic fixture accessor for fixture names that may conflict.
+      #
+      #   assert_equal "Ruby on Rails", web_sites(:rubyonrails).name
+      #   assert_equal "Ruby on Rails", fixture(:web_sites, :rubyonrails).name
```

Comment:
I'm an RDoc noob, definitely let me know if I'm missing something. Perhaps point the source to `_active_record_fixture`? Not sure how to do that.